src/securemul.rs

```diff
@@ -191,6 +191,9 @@ pub mod stream {
         /// of a `Stream`.
         #[tokio::test]
         async fn supports_stream_of_secret_shares() {
+            // https://github.com/rust-lang/rfcs/issues/1664
```

Comment:
could you add a line quickly summarizing the issue? something like "need to setup logging inside #[test] block".